src/sentry/api/endpoints/organization_releases.py

```diff
@@ -74,6 +86,42 @@ class ReleaseSerializerWithProjects(ReleaseWithVersionSerializer):
     refs = ListField(child=ReleaseHeadCommitSerializer(), required=False, allow_null=False)
 
 
+def debounce_update_release_health_data(organization, project_ids):
+    """This causes a flush of snuba health data to the postgres tables once
+    per minute for the given projects.
+    """
+    # Figure out which projects need to get updates from the snuba.
+    should_update = {}
+    cache_keys = ["debounce-health:%d" % id for id in project_ids]
```

Comment:
I wonder if it would be cheaper to have a single debounce key for the entire org that contains a list of project IDs.

Reply:
I want to revisit this entire upserting. Ideally this could all happen in a kafka consumer.